rosetta-ide/src/main/java/com/regnosys/rosetta/ide/semantictokens/RosettaSemanticTokensService.java

```diff
@@ -263,7 +263,7 @@ public SemanticToken markFeature(RosettaFeatureCall featureCall) {
 	
 	private SemanticToken markSymbol(EObject objectToMark, EStructuralFeature featureToMark, RosettaSymbol symbol) {
 		if (symbol instanceof Attribute) {
-			RType implicitType = typeProvider.typeOfImplicitVariable(objectToMark);
+			RType implicitType = typeProvider.typeOfImplicitVariable(objectToMark).getRType();
 			if (implicitType != null) {
 				Set<? extends RosettaFeature> implicitFeatures = Sets.newHashSet(extensions.allFeatures(implicitType, objectToMark));
```

Comment:
This will need to be adapted. It could also have implicit meta features.

Reply:
The method `public SemanticToken markMetaMemberInAnnotation(AnnotationRef annotation)` is already handling this

Reply:
Think I've fixed this, you should check out my approach and see if you agree

Reply:
Have backed out `metaType` changes as discussed